src/scheduler/constant.h

```diff
@@ -521,7 +521,8 @@ enum nodepart
 {
 	NP_LOW = 0,
 	NP_IGNORE_EXCL = 1,
-	NP_CREATE_REST = 2
+	NP_CREATE_REST = 2,
+	NP_NO_CACHE_UPDATE = 4
```

Comment:
This isn't a cache.  It's an array of node partitions it is in.  The cache is when we create node partitions on the fly . It is stored elsewhere.  This should be something like NP_NO_ADD_NP_NODE_ARR or something like that.